File: sol/solution/8 Kyu/572b77262bedd351e9000076/Solution.py
#
# Solution
#
def first(seq, n=1):
    if n == 0:
        return []
    return seq[:n]





#
# Sample Tests
#
@test.describe('Example Tests')
def example_tests():
    seq = ['a', 'b', 'c', 'd', 'e']
    test.assert_equals(first(seq), ['a'])
    test.assert_equals(first(seq, 0), []);
    test.assert_equals(first(seq, 1), ['a']);
    test.assert_equals(first(seq, 2), ['a', 'b']);
    test.assert_equals(first(seq, 10), ['a', 'b', 'c', 'd', 'e'])

File: sol/solution/8 Kyu/572b77262bedd351e9000076/Solution.js
/*
Solution
*/
function first(arr, n = 1) {
  if (n === 0) return [];
  return arr.slice(0, n);
}





/*
Sample Tests
*/
const { assert } = require('chai');

describe("Tests", () => {
  it("test", () => {
    const arr = ['a', 'b', 'c', 'd', 'e'];
    assert.sameOrderedMembers(first([...arr]   ),      ['a']);
    assert.sameOrderedMembers(first([...arr], 2), ['a', 'b']);
  });
});
